docs/Examples/spv-sync-wallet.js

```diff
@@ -1,51 +1,78 @@
 'use strict';
 
 const bcoin = require('../..');
-
-bcoin.set('testnet');
+bcoin.set('regtest');
 
 // SPV chains only store the chain headers.
-const chain = new bcoin.Chain({
-  memory: false,
-  location: '/tmp/bcoin/spvchain',
+const SPVchain = new bcoin.Chain({
   spv: true
 });
-
-const pool = new bcoin.Pool({
-  chain: chain,
-  spv: true,
-  maxPeers: 8
+const SPVpool = new bcoin.Pool({
+  chain: SPVchain,
+  maxOutbound: 1
 });
+const SPVwalletdb = new bcoin.wallet.WalletDB({ memory: true });
 
-const walletdb = new bcoin.wallet.WalletDB({ memory: true });
+// FULL node will provide tx data to SPV node
+const FULLchain = new bcoin.Chain();
```

Comment:
`fullchain` and `spvchain` ?

Reply:
This script spawns two separate nodes, one full one spv. Further down they connect on localhost and a tx is transmitted from full to spv.

Reply:
I was referring to the formatting.

Reply:
Ah, ha yeah I'm conflicted because SPV is an all-caps acronym, but then I want FULL to match that style so it's super duper clear whats what